launchpad/launch/run_locally/launch_local_output_to_files.py

```diff
@@ -32,6 +36,9 @@ def launch_and_output_to_files(commands_to_launch):
   titles = []
   manager = worker_manager.WorkerManager()
   atexit.register(manager.wait)
+  print('Logs are being output to: {}. '
+        'The logging directory can be customized by setting the '
+        'LAUNCHPAD_LOGGING_DIR environment variable.'.format(_LOGGING_DIR))
```

Comment:
Nit: consider using f-string instead? https://docs.python.org/3/tutorial/inputoutput.html#formatted-string-literals

Reply:
Sure. Just wanted to be consistent with the other print statement in that function and used the old style `.format`. Changed both to f-string.